### PR TITLE
Log block import source

### LINKED ISSUE #5721: Track block source on import log / metric
## Description

When debugging it's useful to know if an imported block was sourced from gossip / lookup / range sync. This information can be ultimately deduced by inspecting logs, but it's time-consuming. It would be just simpler to append a `source` property in the import log.

### PR DESCRIPTION
**Pull request by eserilev**
## Issue Addressed

Closes #5721

## Proposed Changes

Add a source tag to block import logs to indicate where the imported block came from. Options are Lookup, Gossip, RangeSync and HttpApi

### CODE REVIEW

**Comment by eserilev** [comment]
image
example logs

File: beacon_node/beacon_chain/src/beacon_chain.rs [hunk 1]
```diff
@@ -3048,6 +3055,7 @@ impl<T: BeaconChainTypes> BeaconChain<T> {
                     "Beacon block imported";
                     "block_root" => ?block_root,
                     "block_slot" => block_slot,
+                    "source" => block_source.to_string(),
```
Comment (dapplion) [inline beacon_node/beacon_chain/src/beacon_chain.rs]: You should able to do this if you implement display
```
"source" => %block_source,
```

**Review by dapplion — APPROVED** [review]
Thank you for implementing this!

**Comment by jmcph4** [comment]
> image example logs

Love this!

**Comment by realbigsean** [comment]
@mergify queue

**Pull request merged**